docs/modeling/connectors/ootb/content.md

```diff
@@ -0,0 +1,367 @@
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `folders` | *Required.* The folder returned as a [process variable](../../processes/variables.md) of type folder. | Folder | 2020-shipments | 
+
+### Select content metadata
+The `SELECT_METADATA` action is used to select the metadata from a file or folder from Alfresco Content Services.
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `useFiles` | *Optional.* Set to `true` to include files in the search query.  | Boolean | True |
+| `useFolders`| *Optional.* Set to `true` to include folders in the search query. | Boolean | False | 
+
+The following is an example response of the selected metadata:
+
+```json
+{
+	"164e0082-5ac1-4afa-a09b-619a8c79f66f": {
+		"cm:categories": [],
+		"cm:lastThumbnailModification": [
+			"imgpreview:1588671895595"
+    ]
+  }
+}
+```
+
+## Events
+The following events can be configured as event [triggers](../../triggers.md) using the content connector:
+
+* [Content created](#content-created)
+* [Content updated](#content-updated)
+* [Content moved](#content-moved)
+* [Content deleted](#content-deleted)
+
+### Content created
+This event is fired when a file or folder is created in Alfresco Content Services that matches the input parameters defined in the trigger event.
+
+For a file the event is called `FILE_CREATED`. 
+
+For a folder the event is called `FOLDER_CREATED`.
+
+#### Input parameters
+
+| Parameter | Description | Type | Example |
+| --------  | ----------- | ---- | ------- |
+| `nodeName` | The name of the node that is created. | String | 2020-orders | 
+| `parentFolder` | The node ID of the parent folder. | String | dioos2a1-2a22-4f42-820d-c2b628751121 | 
+| `nodeType` | The content type of the node that is created. | String | cm:file | 
+
+#### Output parameters
+
+| Parameter | Description | Type | Example |
+| --------  | ----------- | ---- | ------- |
+| `type` | The Java class of the resource. | String | |
+| `id` | The node ID of the node that is created. | String | dioos2a1-2a22-4f42-820d-c2b628751121 | 
+| `nodeType` | The content type that is created. | String | cm:file | 
+| `isFile` | A flag for whether the node is a file. | Boolean | True | 
+| `isFolder` | A flag for whether the node is a folder. | Boolean | False | 
+| `primaryHierarchy` | An array of parent nodes | Array | ioos2a1-2a22-4f42-820d-c2b628751121, 164e0082-5ac1-4afa-a09b-619a8c79f66f | 
+| `properties` | The properties of the created node. | JSON | |
+| `affectedPropertiesBefore` | The properties of the node before the event. | JSON | | 
+| `affectedPropertiesAfter` | The properties of the node after the event. | JSON | | 
+| `aspects` | The aspects of the created node. | Array | | 
+| `aspectNamesBefore` | A list of the aspect names before the event. | Array | versionable | 
```

Comment:
aspects are returned with their prefixed name, for example `cm:versionable` (same below)

Reply:
Updated